pytential/symbolic/stokes.py

```diff
@@ -123,12 +123,12 @@ def apply(self, density_vec_sym, mu_sym, qbx_forced_limit):
                 ctr_key = tuple(var_ctr)
 
                 if i < 1:
-                    sym_expr[comp] = sym.IntG(
+                    sym_expr[comp] = sym.S(
```

Comment:
I find this notationally weird, given that these are not single layer potentials really. I think we should have an `IntG`-like thing that vectorizes over densities. (and not use `S`)